.github/workflows/ci_steps.yml

```diff
@@ -178,7 +178,12 @@ jobs:
         if: inputs.msystem != ''
         run: |
           cmake --version
-          cmake $CMAKE_ARGS
+          if [ "${{ inputs.msystem }}" == "MINGW32" ]; then
```

Comment:
You probably tried to put that with the block above but got stymied by the quoting. I'll come up with a better way of collecting the arguments, so it preserves arguments with spaces.

Reply:
yes, exactly - thought single quotes would work, then tried escaping, not sure what i was doing wrong...